core/src/main/java/bio/terra/pearl/core/service/workflow/TriggerActionService.java

```diff
@@ -50,12 +63,30 @@ public void handleEvent(EnrolleeEvent event) {
             if (TriggerActionType.NOTIFICATION.equals(trigger.getActionType())) {
                 notificationDispatcher.dispatchNotificationAsync(trigger, event.getEnrolleeContext(),
                         event.getPortalParticipantUser().getPortalEnvironmentId());
+            } else if (TriggerActionType.ADMIN_NOTIFICATION.equals(trigger.getActionType())) {
+                sendEmailToAllAdmins(trigger, event);
             } else if (TriggerActionType.TASK_STATUS_CHANGE.equals(trigger.getActionType())) {
                 updateTaskStatus(trigger, event);
             }
         }
     }
 
+    private void sendEmailToAllAdmins(Trigger trigger, EnrolleeEvent event) {
+        Portal portal = portalService
+                .findByPortalEnvironmentId(event.getPortalParticipantUser().getPortalEnvironmentId())
+                .orElseThrow(() -> new IllegalStateException("Portal not found"));
+
+        EmailTemplate emailTemplate = emailTemplateService.find(trigger.getEmailTemplateId())
+                .orElseThrow(() -> new NotFoundException("Email template not found"));
+        emailTemplateService.attachLocalizedTemplates(emailTemplate);
+
+        NotificationContextInfo contextInfo = adminEmailService.loadContextInfoFromEnrollee(emailTemplate, portal, event.getEnrolleeContext().getEnrollee().getStudyEnvironmentId());
+
+        List<AdminUser> adminUsers = adminUserService.findAllWithRolesByPortal(portal.getId());
+        for (AdminUser adminUser : adminUsers) {
```

Comment:
SendGrid should support sending email that has multiple cc'ed recipients, and that will save us this loop (although might require refactoring some of the underlying methods to take a List). 

We definitely should do that, although it's fine if that's in a separate PR